sdkproject/Assets/Mapbox/Unity/SourceLayers/ImageryLayer.cs

```diff
@@ -110,10 +109,10 @@ public void Initialize()
 			_imageFactory.SetOptions(_layerProperty);
 
 			//updating image layer on settings change
-			_layerProperty.OnPropertyUpdated += Refresh;
+			_layerProperty.PropertyHasChanged += UpdateImageryLayer;
 		}
 
-		public void Refresh()
+		public void UpdateImageryLayer(object sender, System.EventArgs e)
```

Comment:
Can we rename this to be UpdateLayer or RedrawLayer - so that the signatures for all layer updates are similar.